<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="imageDownloadButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>530</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Image/Weather Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>431</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="currentGroupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 178, 135);
background-color:rgb(2,2,2);</string>
    </property>
    <property name="title">
     <string>Current Time</string>
    </property>
    <widget class="QLCDNumber" name="secondLCD">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>22.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>22</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="minuteLCD">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>22.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>22</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="hourLCD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>22.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>22</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="worldGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 178, 135);
background-color:rgb(2,2,2);</string>
    </property>
    <property name="title">
     <string>World Clock</string>
    </property>
    <widget class="QLCDNumber" name="worldSecondLCD">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>22.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>22</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="worldMinuteLCD">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>22.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>22</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="worldHourLCD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>22.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>22</number>
     </property>
    </widget>
    <widget class="QComboBox" name="worldComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtBottom</enum>
     </property>
     <item>
      <property name="text">
       <string>America, Chicago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>America, New York</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Japan, Tokyo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Korea, Seoul</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thailand, Bangkok</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Israel, Jerusalem</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>China, Shanghai</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mali, Bamako</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Senegal, Dakar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spain, Madrid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>England, London</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Italy, Rome</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="weatherLabel_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>weatherLabel_2</zorder>
    <zorder>worldSecondLCD</zorder>
    <zorder>worldMinuteLCD</zorder>
    <zorder>worldHourLCD</zorder>
    <zorder>worldComboBox</zorder>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>530</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Please enter ZIP code</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:rgb(220, 220, 220)</string>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>590</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 178, 135)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>281</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 178, 135);</string>
    </property>
    <property name="title">
     <string>To-Do List</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(226, 226, 226);
background-color:rgb(34, 34, 34);</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="greetLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poor Richard</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 178, 135)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weatherPhotoLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>590</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="photoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>591</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(://images/image1.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(2,2,2);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>imageDownloadButton</zorder>
   <zorder>imageLabel</zorder>
   <zorder>currentGroupBox</zorder>
   <zorder>worldGroupBox</zorder>
   <zorder>zipCodeEdit</zorder>
   <zorder>weatherLabel</zorder>
   <zorder>groupBox</zorder>
   <zorder>greetLabel</zorder>
   <zorder>weatherPhotoLabel</zorder>
   <zorder>photoLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpen_To_Do_List"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_To_Do_List">
   <property name="text">
    <string>Open To-Do List</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
